plugin/grpctrace/interceptor_test.go

```diff
@@ -0,0 +1,128 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package grpctrace
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"google.golang.org/grpc"
+
+	"go.opentelemetry.io/otel/api/global"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+type testExporter struct {
+	spanMap map[string][]*export.SpanData
+}
+
+func (t *testExporter) ExportSpan(ctx context.Context, s *export.SpanData) {
+	var spans []*export.SpanData
+	var ok bool
+
+	if spans, ok = t.spanMap[s.Name]; !ok {
```

Comment:
You can replace this function with a one-liner:

```
t.spanMap[s.Name] = append(t.spanMap[s.Name], s)
```

This works because of the way nils are handled in Go.

Reply:
That's actually pretty cool. Thanks for the suggestion!